chmodzip.d

```diff
@@ -26,14 +26,13 @@ int main(string[] args)
             zr.expand(de);
             writefln("name = %s", de.name);
             writefln("\tcomment = %s", de.comment);
-            writefln("\tmadeVersion = x%04x", de.madeVersion);
             writefln("\textractVersion = x%04x", de.extractVersion);
             writefln("\tflags = x%04x", de.flags);
             writefln("\tcompressionMethod = %d", de.compressionMethod);
             writefln("\tcrc32 = x%08x", de.crc32);
             writefln("\texpandedSize = %s", de.expandedSize);
             writefln("\tcompressedSize = %s", de.compressedSize);
-            writefln("\teattr = %03o, %03o", de.externalAttributes >> 16, de.externalAttributes & 0xFFFF);
+            writefln("\teattr = %03o, %03o", de.fileAttributes);
```

Comment:
The public [fileAttributes](https://github.com/dlang/phobos/blob/master/std/zip.d#L162) does this in the setter